env.example.yml

```
@@ -1,17 +1,5 @@
 LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as the subnet)
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
-GEOCODER_API_KEY: INSERT API KEY HERE
-GEOCODE_EARTH_URL: https://api.geocode.earth/v1 # Not needed if geocode.earth is not used
-CSV_ENABLED: true
-CUSTOM_PELIAS_URL: http://<insert your Pelias endpoint here>/v1
-GEOCODER: HERE # Options: HERE, PELIAS
-TRANSIT_GEOCODER: (optional) OTP/PELIAS
-TRANSIT_BASE_URL: (conditionally required) OTP instance when TRANSIT_GEOCODER=OTP (/otp/routers/{routerId})
-
-SECONDARY_GEOCODER: (optional) HERE/PELIAS
-SECONDARY_GEOCODER_API_KEY: (optional) INSERT SECONDARY API KEY HERE
-SECONDARY_GEOCODE_EARTH_URL: (optional) https://api.geocode.earth/v1 # Not needed if geocode.earth is not used
-
-REDIS_HOST: (optional) <insert IP of redis host here>
-REDIS_KEY: (optional) <insert redis password here>
+GEOCODERS: <JSON Array of OTP-UI `GeocoderConfig`s>
+BACKUP_GEOCODERS: <JSON Array of OTP-UI `GeocoderConfig`s. Same length as GEOCODERS> 
```

Are backup geocoders supposed to be in the same order as the main ones? Is each backup geocoder at a given index supposed to be of the same kind as the main one? If so, would it make sense to write something like:
```
GEOCODERS:
- main:
    ...
  backup:
    ...
- main:
    ...

---

That would make sense but we're dealing with serverless which loads all these in as env variables. Doing what you suggest is technically possible but would be a huge overhead to implement

---

Same thing, just tweak the comment:
```suggestion
BACKUP_GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`s. Same length and order as GEOCODERS> 
```